### PR TITLE
Docs for PowerShell Core containers on ARM

### PR DESCRIPTION
**Pull request by pcgeek86**
# PR Summary

Add guidance for building and running PowerShell Core containers on ARM-powered devices, such as Raspberry Pi.

## PR Context

Select the type(s) of documents being changed.

**Cmdlet reference & about_ topics**
- [ ] Version 7 content
- [ ] Version 6 content
- [ ] Version 5.1 content
- [ ] Version 5.0 content
- [ ] Version 4.0 content
- [ ] Version 3.0 content

**Conceptual articles**
- [x] Fundamental conceptual articles
- [ ] Script sample articles
- [ ] DSC articles
- [ ] Gallery articles
- [ ] JEA articles
- [ ] WMF articles
- [ ] SDK articles

## PR Checklist

- [x] I have read the [contributors guide](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/CONTRIBUTING.md) and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [x] Includes content related to issues and PRs - see [Closing issues using keywords](https://help.github.com/en/articles/closing-issues-using-keywords).
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the
    title and remove the prefix when the PR is ready.

### CODE REVIEW

**Comment by sdwheeler** [comment]
@TravisEz13 Can you review and comment on this?

File: reference/docs-conceptual/install/PowerShell-Core-on-ARM.md [hunk 1]
```diff
@@ -13,6 +13,20 @@ For more information on installing these experimental builds, see the installati
 [Windows 10 IoT](installing-powershell-core-on-windows.md#deploying-on-windows-iot)
 and [Raspbian](installing-powershell-core-on-linux.md#raspbian).
 
+## PowerShell Core in Docker
+
+You can run PowerShell Core in a Docker container on a ARM device, such as a Raspberry Pi. There is not currently a pre-built container image published to a public repository, such as Docker Hub. However, you can build your own container image by downloading the `Dockerfile` for [PowerShell Core on ARM](https://github.com/PowerShell/PowerShell-Docker/blob/master/release/stable/arm32v7/docker/Dockerfile).
```
Comment (TravisEz13) [inline reference/docs-conceptual/install/PowerShell-Core-on-ARM.md]: It looks like we can build this image when I checked today.

I filed an issue for this: https://github.com/PowerShell/PowerShell-Docker/issues/296

This shouldn't be hard to do.  I'll ping some of our contributors and see if I can get them to do this.

**Comment by sdwheeler** [comment]
Holding this PR unmerged for now. @TravisEz13 is working on creating a Docker image for ARM. When that is complete this PR will not be needed.

File: reference/docs-conceptual/install/PowerShell-Core-on-ARM.md (hunk 1 above)
Comment (TravisEz13) [reply reference/docs-conceptual/install/PowerShell-Core-on-ARM.md]: `docker pull mcr.microsoft.com/powershell:preview-arm32v7-ubuntu-18.04`

See docker hub for other tags, only preview is currently available:
image

Comment (TravisEz13) [reply reference/docs-conceptual/install/PowerShell-Core-on-ARM.md]: https://hub.docker.com/_/microsoft-powershell

**Comment by sdwheeler** [comment]
You can use the docker images that @TravisEz13 mentioned with the existing instructions. Closing this PR as unneeded.

**Pull request closed without merging**